src/main/kotlin/com/zepben/evolve/cim/iec61968/common/StreetAddress.kt

```diff
@@ -12,6 +12,8 @@ package com.zepben.evolve.cim.iec61968.common
  *
  * @property postalCode Postal code for the address.
  * @property townDetail Town detail.
+ * @property poBox Post office box.
+ * @property streetDetail Street detail.
  */
-data class StreetAddress(val postalCode: String, val townDetail: TownDetail?)
+data class StreetAddress(val postalCode: String, val townDetail: TownDetail?, val poBox: String, val streetDetail: StreetDetail?)
```

Comment:
Can you please add some default values to these as they are no longer all needed to make an address and it makes it much nicer to use (I am sure you noticed when having to update a heap of tests with default values)

Reply:
Like this?
```
data class StreetAddress(
    val postalCode: String = "1234",
    val townDetail: TownDetail? = TownDetail("name", "stateOrProvince"),
    val poBox: String = "5678",
    val streetDetail: StreetDetail? = StreetDetail("buildingName", "floorIdentification", "name", "number", "suiteNumber", "type", "displayAddress")
)
```

Reply:
New revision uses above code.